<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Record Audio</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>418</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QPushButton" name="stopRecordButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Stop Recording</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="durationLabel">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(185, 185, 185);</string>
      </property>
      <property name="text">
       <string>00:00:00:000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="startRecordButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Start Recording</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="auidoInputLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(185, 185, 185);</string>
      </property>
      <property name="text">
       <string>Audio device:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="audioInputComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="maskLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(26, 30, 34, 129);
color: rgb(185, 185, 185);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
